konduit-serving-codegen/src/test/java/JsonCoverageTest.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
konduit-serving-codegen is an "old api" (to be deleted) module so this one can go